MARC.js

```diff
@@ -541,7 +542,26 @@ record.prototype.translate = function(item) {
 		this._associateNotes(item, "545", "ab");
 		
 		// Extract title
-		this._associateDBField(item, "245", "abn", "title");
+		// a = main title
+		// b = subtitle
+		// n = volume title
+		var titlesubfields = this.getFieldSubfields("245")[0];
+		item.title = titlesubfields["a"];
+		if (titlesubfields["b"]) {
+			//If the title did not end in a punctuation mark, we should add a colon
+			if (item.title.search(/[\?:,\.!;]\s*$/) == -1) {
+				item.title += ':';
+			}
+			item.title += ' ' + titlesubfields["b"];
+		}
+		if (titlesubfields["n"]) {
```

Comment:
See https://github.com/zotero/translators/pull/847#discussion_r24299386

But maybe we should save subfield "$p - Name of part/section of a work" as well, because that is the reason to not use the volume (number) but extending it into the title field. What do you think?